theme/greenscreen.less

```diff
@@ -35,6 +35,7 @@
 
     .monaco-editor .lines-content, .monaco-editor .view-line, .monaco-editor .view-lines {
         text-shadow: 1px 1px #000;
+        font-weight: 500;
```

Comment:
hm claires suggestion sounds somewhat like where we started, something like

![image](https://user-images.githubusercontent.com/5615930/183125897-20b2def5-db91-4403-a6a2-c974961a9335.png)

```suggestion
        font-weight: 500;
        > span {
            background: @editorToolsBackground;
        }
```

(plus deleting the text shadow on the previous line too, but I don't think the suggestion feature allows making suggestions on non-modified lines)

Reply:
Sounds good, this makes sense!